src/components/Announce/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import strings from 'lang';
+
+import blogpost from './blogpost';
+import styles from './styles.css';
+
+const BlogpostAnnounce = () => {
```

Comment:
are you planning to create a different-looking announcement mechanism for releases and blog posts?  maybe we can just use the same one?